atlas-shell-tools/README.md

```diff
@@ -1,23 +1,53 @@
 # Atlas Shell Tools
-A command line interface for `atlas`
+A command line interface for [`osmlab/atlas`](https://github.com/osmlab/atlas) (and downstream repositories) tools.
 
+##### Table of Contents
+1. [What are the Atlas Shell Tools?](#whatare)
+2. [Installation](#install)
+   * [Auto-install for `bash` users](#bashinstall)
+   * [Auto-install for `zsh` users](#zshinstall)
+   * [Manual install](#manualinstall)
+3. [Managing Your Installation](#managing)
+   * [Installing Modules](#moduleinstall)
+   * [Switching The Active Module](#moduleswitch)
+   * [Viewing Command Documentation](#viewdocs)
+   * [Saving Command Option Presets](#savepresets)
+   * [And Much More](#andmuchmore)
+4. [Creating Your Own Command](#creating)
+5. [Updating The Commands And Tools](#updating)
+
+<a name="whatare"/>
```

Comment:
`<a name="whatare"></a>`  <-- This closing tag should fix the formatting issues @lucaspcram